environments/dev-env/README.md

```diff
@@ -0,0 +1,395 @@
+      spec:
+        cells:
+          foo: bar
+      EOF
+      ```
+
+   * Step-3: create a Location object describing your edge pcluster. For example:
+
+      ```bash
+      cat <<EOF | kubectl apply -f -
+      apiVersion: scheduling.kcp.io/v1alpha1
+      kind: Location
+      metadata:
+        name: location-f
+        labels:
+          env: prod
+      spec:
+        resource: {group: workload.kcp.io, version: v1alpha1, resource: synctargets}
+        instanceSelector:
+          matchLabels: {"example":"si", "extended":"non"}
+      EOF
+     ```
+
+      A location and synctarget objects will be created:
+
+      ```bash
+      kubectl get locations,synctargets
+      NAME                                    RESOURCE      AVAILABLE   INSTANCES   LABELS   AGE
+      location.scheduling.kcp.io/default      synctargets   0           1                    36s
+      location.scheduling.kcp.io/location-f   synctargets   0           1                    25s
+
+      NAME                                       AGE
+      synctarget.workload.kcp.io/sync-target-f   36s
+      ```
+
+      The [mailbox-controller](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/mailbox-controller/) creates a mailbox workspace for the newly created SyncTarget: `sync-target-f`:
+
+      ```bash
+      kubectl ws root
+      Current workspace is "root".
+
+      kubectl ws tree
+      .
+      └── root
+          ├── compute
+          ├── espw
+          │   └── 1q1p9rsh18rhjuy4-mb-2c1b6ce7-bc4a-4071-887d-871ba293f303
+          ├── imw-1
+          └── my-org
+              └── wmw-1
+      ```
+
+* Populate the `wmw`: 
+
+  * Step-1: Enter the target workspace: `wmw-1`
+ 
+    ```bash
+      kubectl ws root:my-org:wmw-1
+    ```
+
+  * Step-2: create your workload objects. For example:
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: commonstuff
+      labels: {common: "si"}
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: nginx-deployment
+      labels:
+        app: nginx
+    spec:
+      replicas: 3
+      selector:
+        matchLabels:
+          app: nginx
+      template:
+        metadata:
+          labels:
+            app: nginx
+        spec:
+          containers:
+          - name: nginx
+            image: nginx:1.14.2
+            ports:
+            - containerPort: 80
+    EOF
+    ```
+
+  * Step-3: create the `EdgePlacement` object for your workload. Its “where predicate” (the locationSelectors array) has one label selector that matches the Location object created earlier, thus directing the workload to your edge pcluster.
+ 
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: edge.kcp.io/v1alpha1
+    kind: EdgePlacement
+    metadata:
+      name: edge-placement-c
+    spec:
+      locationSelectors:
+      - matchLabels: {"env":"prod"}
+      namespaceSelector:
+        matchLabels: {"common":"si"}
+    EOF
```

Comment:
You also need this EdgePlacement to call for the downsync of the APIBinding to the kubernetes APIs.